packages/block-editor/src/components/block-mover/style.scss

```diff
@@ -78,3 +78,11 @@
 .block-editor-block-mover__description {
 	display: none;
 }
+
+.block-editor-block-mover {
+	&.is-horizontal {
+		.block-editor-block-mover__control-drag-handle {
```

Comment:
Since there are no other styles that are decedents of any of these selectors, feel free to combine them in one line

Reply:
Done!